src/index.js

```diff
@@ -49,7 +49,9 @@ export default function configureStore(middlewares=[]) {
           if (isFunction(cb)) {
             listeners.push(cb);
           }
-          return null;
+          const index = listeners.indexOf(cb)
+          if (index < 0) { throw new Error('couldn\'t find callback in array') }
```

Comment:
Why are you throwing an error here? 

In the official implementation, it doesn't throw an error. https://github.com/reactjs/redux/blob/b523aa4ae52f7afcbc66276cd18d5dc226c30b45/src/createStore.js#L101